### PR TITLE
Refactoring/fixes after code style review

### PR DESCRIPTION
**Pull request by xahgmah**

### CODE REVIEW

File: video_xblock/static/html/studio-edit-tab.html [hunk 1]
```diff
@@ -15,25 +15,19 @@
...
                 {% include "fields/string.html" with field=field %}
 
             {% elif field.type == "integer" or field.type == "float" %}
-                <span>number</span>
                 {% include "fields/number.html" with field=field %}
 
-            {% elif field.type == "text" or field.type == "html" %}
-                <span>text</span>
-                {% include "fields/text.html" with field=field %}
+            {% elif field.type == "text" or field.type == "html" or field.type == 'generic' or field.type == 'list' or field.type == 'set' %}
```
Comment (z4y4ts) [inline video_xblock/static/html/studio-edit-tab.html]: Can it be `{% elif field.type in ('text, 'html', ..., 'set') %}`?

File: video_xblock/static/js/studio-edit-utils.js [hunk 2]
```diff
@@ -31,25 +31,21 @@ function fillValues(fields) {
...
  * Display message with results of a performed action (e.g. a transcript manual or automatic upload).
  */
-function showStatus(message, type, successMessageElement, errorMessageElement) {
+function showStatus(message, type, statusElement) {
     'use strict';
-    var elementToEmpty = '';
-    var elementToShow = '';
-    var SUCCESS = 'success';
-    var ERROR = 'error';
+    var hideIn = 5000;
```
Comment (z4y4ts) [inline video_xblock/static/js/studio-edit-utils.js]: Please add inline comment `// 5 seconds`

File: video_xblock/backends/youtube.py [hunk 3]
```diff
@@ -178,10 +169,10 @@ def format_transcript_timing(sec, period_type=None):
             period_type (str): Timing period type (whether `end` or `start`).
         """
         # Get rid of overlapping periods.
-        if period_type == 'end' and float(sec) >= 0.001:
-            float_sec = float(sec) - 0.001
-            sec = float_sec
-        mins, secs = divmod(sec, 60)  # pylint: disable=unused-variable
+        float_sec = float(sec)
+        sec = float_sec - 0.001 if period_type == 'end' and float_sec >= 0.001 else sec
```
Comment (z4y4ts) [inline video_xblock/backends/youtube.py]: This ternary operator is rather hard to comprehend.
Also, still unclear what's the fuzz with 0.001?

File: video_xblock/static/js/studio-edit-transcripts-autoload.js [hunk 4]
```diff
@@ -223,20 +210,19 @@ function removeEnabledTranscriptBlock(enabledTranscript, initialDefaultTranscrip
     isStoredVideoPlatform = $.inArray(langCode, initialDefaultTranscriptsLangCodes) !== -1;  // Is in array
     // Display message with results of removal
     if (isSuccessfulRemoval && isStoredVideoPlatform) {
-        message = successMessageRemoval;
+        message = langLabel + ' transcripts are successfully removed from the list of enabled ones.';
```
Comment (z4y4ts) [inline video_xblock/static/js/studio-edit-transcripts-autoload.js]: What about i18n here?

File: video_xblock/static/js/studio-edit-utils.js [hunk 5]
```diff
@@ -31,25 +31,21 @@ function fillValues(fields) {
 /**
  * Display message with results of a performed action (e.g. a transcript manual or automatic upload).
  */
-function showStatus(message, type, successMessageElement, errorMessageElement) {
+function showStatus(message, type, statusElement) {
```
Comment (z4y4ts) [inline video_xblock/static/js/studio-edit-utils.js]: Please make it `$statusElement` to explicitly state jQuery object is expected.

File: video_xblock/static/js/studio-edit.js [hunk 6]
```diff
@@ -523,7 +521,7 @@ function StudioEditableXBlock(runtime, element) {
      */
     function bindRemovalListenerEnabledTranscript(langCode, langLabel, downloadUrlServer) {
         var $removeElement = $('.default-transcripts-action-link.remove-default-transcript[data-lang-code=' + langCode + ']');
-        $removeElement.click(function() {
+        $removeElement.click(function(e) {
```
Comment (z4y4ts) [inline video_xblock/static/js/studio-edit.js]: Single-char var

File: video_xblock/static/css/student_view.css [hunk 7]
```diff
@@ -1,3 +1,6 @@
+.is-hidden {
```
Comment (z4y4ts) [inline video_xblock/static/css/student_view.css]: For the sake of consistency, please name this file `student-view.css` :)

File: video_xblock/static/html/studio-edit-tab.html (hunk 1 above)
Comment (xahgmah) [inline video_xblock/static/html/studio-edit-tab.html]: No. Need to be written custom templatetag.

File: video_xblock/backends/youtube.py [hunk 8]
```diff
@@ -168,7 +168,8 @@ def format_transcript_timing(sec, period_type=None):
             sec (str): Transcript timing in seconds with milliseconds resolution.
             period_type (str): Timing period type (whether `end` or `start`).
         """
-        # Get rid of overlapping periods.
+        # Youtube returns transcripts with the equal endtime and startime for previous and next transcript blocks
+        # respectively. That is why transcript blocks are overlapping. Get rid of it by decreasing timing on 0.001.
```
Comment (z4y4ts) [inline video_xblock/backends/youtube.py]: 👍

**Pull request merged**